test/data/sample-multipart-oas3.js

```diff
@@ -46,6 +46,12 @@ export default {
                     }
                   }
                 }
+              },
+              encoding: {
+                'email[]': {
+                  style: 'form',
```

Comment:
I have interpreted that if no Encoding Object is set, `explode` will not be enabled by default and the format will be based on the default value of contentType.

https://github.com/OAI/OpenAPI-Specification/blob/v3.1.0-dev/versions/3.1.0.md#encoding-object

So I thought I had to explicitly specify `explode`. (I personally felt that it would be more natural to treat it as `explode` by default ...)